### ISSUE TITLE
[Bug]: Terminal and windows controls are not transparent

### ISSUE THREAD

**Issue by MK-coding** [illixion/vscode-vibrancy-continued#75]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

![vibrancy_bug](https://github.com/illixion/vscode-vibrancy-continued/assets/81474503/a7d3cdba-3b0c-4c9d-9bbd-4b12e8c06a5b)

### Expected Behavior

![theme-solarized-dark+](https://github.com/illixion/vscode-vibrancy-continued/assets/81474503/08f4af14-0866-4ac7-93c9-5488f3bff342)

### Environment

```markdown
- OS: Windows11
- VSCode:1.78.2
- Extension:1.1.15
- Theme:Dark
```

**Comment by MK-coding** [comment]
"terminal.integrated.gpuAcceleration": "off"

**Comment by illixion** [comment]
Yeah, that option from the FAQ should fix your terminal. Window controls aren't transparent due to "native window controls" being enabled by default, though Vibrancy should turn it off automatically.
